theano/gpuarray/basic_ops.py

```diff
@@ -293,6 +293,37 @@ class GpuKernelBase(object):
     """
     params_type = gpu_context_type
 
+    def get_params(self, node):
+        # Default implementation, suitable for most sub-classes.
+        # To be necessarly overridden if a subclass uses a ParamsType
+        # (see GpuCumOp, GpuAdvancedIncSubtensor1_dev20 for examples).
```

Comment:
and `GpuAdvancedIncSubtensor1_dev20.get_params` is still the original implementation.